css/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -37,28 +37,49 @@ main {
 
 .box--top-left {
   background-color: #aed6f1;
+  position: absolute;
+  top: 0;
+  left: 0;
 }
 
 .box--top-center {
   background-color: #85c1e9;
+  position: absolute;
```

Comment:
If you put position: absolute in the general .box class above, you don't have to put it in every single .box class as you did.